css/style.css

```diff
@@ -263,36 +263,27 @@ input[type="checkbox"] {
 /* ----------- UI - Graph ----------- */
 
 .vis {
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  padding: 30px;
-}
-
-.vis > div {
-  display: flex;
-  align-items: flex-start;
-  justify-content: space-between;
-  height: 100%;
+  display: grid;
+  grid-template-rows: 75% 25%;
+  position: relative;
```

Comment:
Per my [comment](https://github.com/electrolyfish/lightbeam-we/pull/112#issuecomment-318718783) on the PR, reverting `vis` is part 1 of 3 of the fix.